src/HostManager.cpp

```diff
@@ -38,39 +38,34 @@
 #include "post_guard.h"
 
 
-bool HostManager::deleteHost( QString hostname )
+bool HostManager::deleteHost(QString hostname)
 {
     qDebug() << "HostManager::deleteHost(" << hostname.toUtf8().constData() << ") INFO: trying to delete host from host pool, getting write lock...";
```

Comment:
Noted! - I _originally_ thought that just doing a "release" build would set that but it does not!